src/tye/ConfigModel/ConfigService.cs

```diff
@@ -14,6 +14,7 @@ public class ConfigService
         public string Name { get; set; } = default!;
         public bool External { get; set; }
         public string? DockerImage { get; set; }
```

Comment:
I was looking for a clean place to log this as a warning deprecated but I just left it (mostly because I was being lazy).